src/chrometrace.rs

```diff
@@ -0,0 +1,151 @@
+// This code is taken from the flamegraph.rs from rbspy
+// https://github.com/rbspy/rbspy/tree/master/src/ui/flamegraph.rs
```

Comment:
I don't see any chrometrace functionality in rbspy - is this license preamble accurate?

Reply:
Ahh, sorry, I originally copied file from flamegraph.rs -- will fix.

Reply:
thanks! 

btw, there is an open issue in rbspy adding this functionality https://github.com/rbspy/rbspy/issues/140 =)